packages/slice-machine/src/features/documentation/MarkdownRenderer/index.tsx

```diff
@@ -29,10 +32,20 @@ const MarkdownCodeBlock = (props: CodeProps) => {
     return null;
   })();
 
+  const onCopy = () => {
+    if (adapter !== undefined) {
+      void telemetry.track({
+        event: "page-type:copy-snippet",
+        framework: adapter,
```

Comment:
Sorry, I just realized also, but the issue says adapter and framework. Here and in the previous event, we are mixing the two :/

Reply:
Ok if you can confirm with Côme before the exact thing we will give since it's not what we have in the issue :/